jenkins/jenkins-check-nodes.sh

```diff
@@ -0,0 +1,132 @@
+WORKSPACE=$(pwd)
+
+blacklist_path="$HOME/workspace/cache/blacklist"
+if [ -d $blacklist_path ]; then
+    mkdir -p $blacklist_path
+fi
+
+nodes_path="$HOME/nodes/"
+email_notification="cms-sdt-logs@cern.ch"
+job_url="${JENKINS_URL}job/nodes-sanity-check/${BUILD_NUMBER}"
+echo $PATHS >> $WORKSPACE/paths.param
+echo $SINGULARITY >> $WORKSPACE/paths.param
+
+function run_check {
+    node=$1
+    SSH_OPTS="-q -o IdentitiesOnly=yes -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -o ServerAliveInterval=60"
+    scp $SSH_OPTS ${WORKSPACE}/cms-bot/jenkins/nodes-sanity-check.sh ${WORKSPACE}/paths.param "cmsbuild@$node:/tmp" || (echo "Cannot scp script" && exit 1)
```

Comment:
do you really need to copy `${WORKSPACE}/paths.param` ? Can we just pass these as argument in next line e.g.
```
ssh $SSH_OPTS cmsbuild@$node 'sh /tmp/nodes-sanity-check.sh $SINGULARITY $PATHS'; exit_code=$?
```

Reply:
Right, it works if I use double quotes to allow for substitution